torch_em/transform/augmentation.py

```diff
@@ -9,18 +14,18 @@
 # TODO RandomElastic3D ?
 
 
-class RandomElasticDeformation(kornia.augmentation.AugmentationBase2D):
```

Comment:
Please don't make the format changes,that makes it really hard to review when things are more complex.